app/views/jobseekers/job_applications/review.html.slim

```diff
@@ -28,13 +28,21 @@
         li = render "jobseekers/job_applications/review/ask_for_support", allow_edit: true
         li = render "jobseekers/job_applications/review/declarations", allow_edit: true
 
-      h3.govuk-heading-l Confirmation
+      h3.govuk-heading-m = t(".confirmation.heading")
 
       = f.govuk_check_boxes_fieldset :confirm_data_accurate, multiple: false, legend: nil do
-        = f.govuk_check_box :confirm_data_accurate, 1, 0, multiple: false, link_errors: true, label: { text: "Confirm data accurate" }
+        = f.govuk_check_box :confirm_data_accurate, 1, 0, multiple: false, link_errors: true, label: { text: t(".confirmation.checkbox_label.confirm_data_accurate") }
```

Comment:
Instead of having a label in the view, you can add a label to the `label` section of `forms.yml`. It'll be something like adding `jobseekers_job_application_review_form` underneath `label`.

Reply:
Same for confirm_data_usage

Reply:
As David points out, these label translations shouldn't be here and we should use the default lookup in the formbuilder to find them